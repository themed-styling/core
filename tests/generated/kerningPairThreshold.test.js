import 'jest-styled-components'

import { kerningPairThreshold } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(kerningPairThreshold, 'kerningPairThreshold')
