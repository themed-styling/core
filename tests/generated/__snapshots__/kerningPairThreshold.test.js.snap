// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kerningPairThreshold renders 0 1`] = `
Array [
  .c0 {
  kerning-pair-threshold: 0;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: 0;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: 0;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`kerningPairThreshold renders function 1`] = `
Array [
  .c0 {
  kerning-pair-threshold: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: has;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: has;
}

<div
    className="c0"
  />,
]
`;

exports[`kerningPairThreshold renders invalid theme path 1`] = `
Array [
  .c0 {
  kerning-pair-threshold: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`kerningPairThreshold renders negative number 1`] = `
Array [
  .c0 {
  kerning-pair-threshold: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`kerningPairThreshold renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    kerning-pair-threshold: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    kerning-pair-threshold: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    kerning-pair-threshold: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    kerning-pair-threshold: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    kerning-pair-threshold: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    kerning-pair-threshold: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    kerning-pair-threshold: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    kerning-pair-threshold: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    kerning-pair-threshold: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    kerning-pair-threshold: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    kerning-pair-threshold: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    kerning-pair-threshold: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    kerning-pair-threshold: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    kerning-pair-threshold: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    kerning-pair-threshold: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    kerning-pair-threshold: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`kerningPairThreshold renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    kerning-pair-threshold: July;
  }
}

@media April {
  .c0 {
    kerning-pair-threshold: August;
  }
}

@media January {
  .c0 {
    kerning-pair-threshold: September;
  }
}

@media April {
  .c0 {
    kerning-pair-threshold: November;
  }
}

@media June {
  .c0 {
    kerning-pair-threshold: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    kerning-pair-threshold: July;
  }
}

@media April {
  .c0 {
    kerning-pair-threshold: August;
  }
}

@media January {
  .c0 {
    kerning-pair-threshold: September;
  }
}

@media April {
  .c0 {
    kerning-pair-threshold: November;
  }
}

@media June {
  .c0 {
    kerning-pair-threshold: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    kerning-pair-threshold: January;
  }
}

@media April {
  .c0 {
    kerning-pair-threshold: December;
  }
}

@media January {
  .c0 {
    kerning-pair-threshold: April;
  }
}

@media April {
  .c0 {
    kerning-pair-threshold: May;
  }
}

@media June {
  .c0 {
    kerning-pair-threshold: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    kerning-pair-threshold: January;
  }
}

@media April {
  .c0 {
    kerning-pair-threshold: December;
  }
}

@media January {
  .c0 {
    kerning-pair-threshold: April;
  }
}

@media April {
  .c0 {
    kerning-pair-threshold: May;
  }
}

@media June {
  .c0 {
    kerning-pair-threshold: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`kerningPairThreshold renders postitive number 1`] = `
Array [
  .c0 {
  kerning-pair-threshold: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`kerningPairThreshold renders string 1`] = `
Array [
  .c0 {
  kerning-pair-threshold: July;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: July;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: June;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: June;
}

<div
    className="c0"
  />,
]
`;

exports[`kerningPairThreshold renders valid theme path 1`] = `
Array [
  .c0 {
  kerning-pair-threshold: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  kerning-pair-threshold: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
